app/Factories/ConfigurationResolverFactory.php

```diff
@@ -32,7 +32,7 @@ class ConfigurationResolverFactory
      */
     public static function fromIO($input, $output)
     {
-        $path = $input->getArgument('path');
+        $path = Project::paths($input);
```

Comment:
Just a stupid grammar thing, but should this variable be called "$paths" instead of "$path" to indicate that it potentially contains multiple paths?

Reply:
Yeah, I didn't change it in an effort to be minimally invasive. Also, the underlying config option for PHP CS Fixer is called `path`. 🤷

Reply:
I agree. Again, just honoring what was already there. My new method name of `paths` may not remain. So it could easily go the other way. 😉